examples/ssh/iperf3.js

```diff
@@ -93,12 +90,12 @@ function checkSshError(err) {
 function executeCommand(commandTemplate, serverIndex, extractorFn) {
     return function (result) {
         var d = D.q.defer();
-        var command = commandTemplate
+        var command = commandTemplate;
 
         if (serverIndex !== null) {
-            if (serverIndex < targetServers.length) {
-                var server = targetServers[serverIndex]
-                command = command.replace("{url}", server.url).replace("{port}", server.port)
+            if (serverIndex < targetServerUrl.length) {
```

Comment:
its still target servers, this is misleading to say the least that the variable is a single URL